test/plugin/scenarios/spring-3.0.x-scenario/src/main/java/test/org/apache/skywalking/apm/testcase/spring3/CaseController.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package test.org.apache.skywalking.apm.testcase.spring3;
```

Comment:
The package name has wrong. `org.apache.skywalking.apm.testcase.*`

And others are ok for me.

Reply:
Using ```org.apache.skywalking``` as the package name,  the ```SpringComponentInstrumentation、SpringRepositoryInstrumentation、SpringServicesInstrumentation```   will not work

https://github.com/apache/skywalking/blob/24cc91c9af6373a97418a011e0d27c166b88782c/apm-sniffer/apm-agent/src/main/java/org/apache/skywalking/apm/agent/SkyWalkingAgent.java#L167-L169

Reply:
[spring-annotation-plugin](https://github.com/apache/skywalking/tree/master/apm-sniffer/optional-plugins/optional-spring-plugins/spring-annotation-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/annotations)

Reply:
i got it. Thanks for your explanation.

Reply:
i got it. Thanks for your explanation.